### PR TITLE
Cg 50 51

### PR DESCRIPTION
**Pull request by cgaddis36**
Created Orders view page for Merchant_employees
-includes each order in the database
-includes each item from that merchant and no other merchants on this show page
Created associating passing test

-cleaned up trailing whitespace in files I was analyzing while working in this code

Also created order show page and appropriating tests to show each item in the order that belongs to the merchant_employees store.  
-Added functionality for fulfilling and unfulfilling an item on a user order

### CODE REVIEW

File: app/views/orders/show.html.erb [hunk 1]
```diff
@@ -58,5 +58,5 @@
   <p> <%= @order.created_at%></p>
 </section>
   <% if (@order.status == "pending") || (@order.status == "packaged") %>
-    <%= link_to "Cancel Order", "/item_orders/#{@order.id}", method: :patch %>
+    <%= link_to "Cancel Order", "/item_orders/#{@order.id}", params: {cancel: "cancel"}, method: :patch %>
   <% end %>
```
Comment (TravisBorgsmiller) [inline app/views/orders/show.html.erb:62]: good use of hidden params

File: app/controllers/merchant/dashboard_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Merchant::DashboardController < Merchant::BaseController
   def index
-    @orders = current_user.merchant.orders.distinct
+    @merchant = current_user.merchant
   end
```
Comment (TravisBorgsmiller) [inline app/controllers/merchant/dashboard_controller.rb:4]: +1

File: app/views/merchant/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+<%= @merchant.name %></br></br>
+<%= @merchant.address %>
+<p><%= @merchant.city %>
+<%= @merchant.state %>
+<%= @merchant.zip %></p>
 
 <section class="grid-container">
 
-<% @orders.each do |order| %>
+<% @merchant.distinct_orders.each do |order| %>
```
Comment (adumortier) [inline app/views/merchant/dashboard/index.html.erb:11]: thank you for adding this, I like that a lot better

File: app/views/orders/show.html.erb [hunk 4]
```diff
@@ -58,5 +58,5 @@
   <p> <%= @order.created_at%></p>
 </section>
   <% if (@order.status == "pending") || (@order.status == "packaged") %>
-    <%= link_to "Cancel Order", "/item_orders/#{@order.id}", method: :patch %>
+    <%= link_to "Cancel Order", "/item_orders/#{@order.id}", params: {cancel: "cancel"}, method: :patch %>
```
Comment (adumortier) [inline app/views/orders/show.html.erb:61]: nice use of hidden params :)

**Review by TravisBorgsmiller — APPROVED** [review]

File: app/controllers/merchant/dashboard_controller.rb [hunk 5]
```diff
@@ -1,6 +1,5 @@
 class Merchant::DashboardController < Merchant::BaseController
   def index
-    @orders = current_user.merchant.orders.distinct
+    @merchant = current_user.merchant
```
Comment (adumortier) [inline app/controllers/merchant/dashboard_controller.rb:3]: I think that looks better too! Thank you for taking the feedback!

**Review by adumortier — APPROVED** [review]

**Pull request merged**